File: 01-Assignment-02-02-2026/Using_C/program_34.c
/* Program 34: Generate random number and check even or odd */

#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int main() {
    srand(time(NULL)); //imp

    int r = rand();
    printf("Random number: %d", r);
    if (r%2 == 0)
        printf("\nRandom No. is Even");
    else
        printf("\nRondom No. is odd");
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program34.java
/*
Program 34
Java program to check whether a generated random number is even or odd
*/

import java.util.Random;

public class Program34 {

    public static void main(String[] args) {
        Random random = new Random();

        int randomInt = random.nextInt(100); // 0 to 99
        System.out.println("Generated random number: " + randomInt);

        if (randomInt % 2 == 0) {
            System.out.println("It is Even");
        } else {
            System.out.println("It is Odd");
        }
    }
}